.changes/slip10-rewrite.md

```diff
@@ -0,0 +1,5 @@
+---
+"iota-crypto": minor
```

Comment:
Should be a patch?

Reply:
Slip10 API is breaking compared to 0.19.

Reply:
It's not. It's minor after all.